src/hardware/design_parameters.rs

```diff
@@ -53,3 +54,7 @@ pub const DDS_SYNC_CLK_DIV: u8 = 4;
 pub const MAX_SAMPLE_BUFFER_SIZE: usize = 32;
 
 pub type SampleBuffer = [u16; MAX_SAMPLE_BUFFER_SIZE];
+
+/// Period for checking the Driver output interlock voltage/current levels.
+pub const DRIVER_MONITOR_PERIOD: Duration<u64, 1, 10000> =
```

Comment:
Maybe better as a miniconf setting?

Reply:
Imo this should not be exposed to a user. I think this should be optimized to run as fast as possible at a later stage.

Reply:
But then it can remain a constant in the app, right? It's not shared anywhere.